src/pages/PlacePage.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="test-page">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <div class = "searchbar">
+      <center>
+      <input class="text-search" type="text" v-model="search" placeholder="Nama Tempat Yang dicari" @change="searching" >
+      <input class="button" type="submit" value ="cari">
+      </center>
+    </div>
+    <div class= "box">
+        <div class = "item" v-for="place in places" :key="place.Id">
+        <header>
+            <img :src="place.Company.Photo.Url" class="photo">
+            <div class = "partner">{{ place.Company.Name }}</div>
+            <div class = "non-rate" v-if="place.Rating == null"> </div>
```

Comment:
```suggestion
            <div class = "non-rate" v-if="!place.Rating"> </div>
```

Shorter condition to check that the value is null